python/ray/workflow/tests/test_dag_to_workflow.py

```diff
@@ -44,6 +44,10 @@ def end(lf, rt, b):
 
 def test_dedupe_serialization_dag(workflow_start_regular_shared):
     from ray.workflow import serialization
+    from ray.workflow.tests.utils import skip_client_mode_test
+
+    # TODO(suquark): Fix workflow with ObjectRefs as inputs under client mode.
+    skip_client_mode_test()
```

Comment:
Is it possible to add the skip as `@pytest.mark.skipif` ?

Reply:
Currently I haven't figured out have to use it in this case - the `use_ray_client` parameter is hidden in the fixture, and we cannot catch it here.